website/content/api-docs/agent/index.mdx

```diff
@@ -281,7 +281,9 @@ This endpoint returns the configuration and member information of the local
 agent. The `Config` element contains a subset of the configuration and its
 format will not change in a backwards incompatible way between releases.
 `DebugConfig` contains the full runtime configuration but its format is subject
-to change without notice or deprecation.
+to change without notice or deprecation. 
+
+~> This endpoint will not reflect [reloaded](/docs/agent/config#reloadable-configuration) configuration values.
```

Comment:
```suggestion
~> This endpoint does not reflect [reloaded](/docs/agent/config#reloadable-configuration) configuration values.
```
Do we mean that the type of information in the response is not within the scope of the reload command? Or that the data is cached in a way that despite issuing the reload command that only the cached data is included in the response?

Reply:
Hi Adam, despite issuing the reload command, only cached data is included in the response.